src/app/page.tsx

```diff
@@ -31,13 +32,25 @@ export default function Home() {
             </div>
           </h2>
           <div className="flex space-x-4">
-          <Link href="/dashboard">
-            <Button className="text-lg rounded-full mt-5 font-bold bg-slate-500 text-white hover:bg-black">Get Started</Button>
+            {/* <Link href="/dashboard">
+              <Button className="text-lg rounded-full mt-5 font-bold bg-slate-500 text-white hover:bg-black">
+                Get Started
+              </Button>
             </Link>
-            <Button className="text-lg rounded-full mt-5 bg-gray-300 text-black hover:text-white hover:bg-black">Read Paper</Button>
+            <Button className="text-lg rounded-full mt-5 bg-gray-300 text-black hover:text-white hover:bg-black">
+              Read Paper
+            </Button> */}
+            <Link href="/create/deployment">
```

Comment:
Let's use just "create" instead of "create/deployment". (But we can leave that for a future PR.)

Reply:
Sure, I have made the change.